src/Sparrow/Logging/Logger.cs

```diff
@@ -48,7 +48,7 @@ public void Operations(FormattableString msg, Exception e = null)
 
         public void Info(string msg, Exception ex = null)
         {
-            _logEntry.At = DateTime.UtcNow;
+            _logEntry.At = DateTime.Now;
```

Comment:
The dates in the logs are `UtcNow` for a reason.
In particular, `UtcNow` is _much_ faster than `Now`.
And this matters a lot in the logs. See:
https://blogs.msdn.microsoft.com/kirillosenkov/2012/01/10/datetime-utcnow-is-generally-preferable-to-datetime-now/